examples/get_sensor_history.py

```diff
@@ -69,48 +69,55 @@ def main():
                        args.start).strftime('%Y-%m-%dT%H:%M:%SZ'),
                    datetime.utcfromtimestamp(args.end).strftime('%Y-%m-%dT%H:%M:%SZ')))
         if len(sensor_names) == 1:
-            # Request history for just a single sensor - result is timestamp, value, status
-            #    If value timestamp is also required, then add the additional argument: include_value_ts=True
-            #    result is then timestmap, value_timestmap, value, status
+            # Request history for just a single sensor - result is
+            # timestamp, value, status
+            #    If value timestamp is also required, then add the additional argument:
+            #        include_value_ts=True
+            #    result is then timestamp, value_timestmap, value, status
+            value_ts = args.include_value_ts
             history = yield portal_client.sensor_history(
-                sensor_names[0], args.start, args.end, timeout_sec=args.timeout)
+                sensor_names[0], args.start, args.end,
+                include_value_ts=value_ts)
             histories = {sensor_names[0]: history}
         else:
             # Request history for all the sensors - result is timestamp, value, status
-            #    If value timestamp is also required, then add the additional argument: include_value_ts=True
+            #    If value timestamp is also required, then add the additional argument:
+            #        include_value_ts=True
             #    result is then timestmap, value_timestmap, value, status
             histories = yield portal_client.sensors_histories(
-                sensor_names, args.start, args.end, timeout_sec=args.timeout)
+                sensor_names, args.start, args.end, include_value_ts=value_ts)
 
         print "Found {} sensors.".format(len(histories))
         for sensor_name, history in histories.items():
             num_samples = len(history)
             print "History for: {} ({} samples)".format(sensor_name, num_samples)
             if num_samples > 0:
-                print "\tindex,timestamp,value,status"
                 for count in range(0, num_samples, args.decimate):
```

Comment:
It will be really good to have an example that shows the use of the averaging sensors, rather than "decimate" (which is much less useful than the averaging case).

Reply:
Please change this to (or add) "averaging" example instead of "decimate" so that users can see an example of the results for averaging queries.

Reply:
Agreed to log a JIRA ticket to add averaging query support to katportalclient